### ISSUE TITLE
[bug] error symlinking web extensions

### ISSUE THREAD

**Issue by Diffussy** [melMass/comfy_mtb#24]

**Comment by melMass** [comment]
Please just read the install instructions 

https://github.com/melMass/comfy_mtb/blob/main/INSTALL.md#web-extensions

**Comment by Diffussy** [comment]
> 

Nice support. Instead of assuming I didn't follow the instructions you should ask. Followed every step and it was working initially just fine then started throwing those errors.

**Comment by melMass** [comment]
Well, while I'm doing my best to provide support and help, sarcasm isn't the best way to kick off a conversation.

You posted an issue with only a title, no body, no explanation of where you come from. So, when I made assumptions based on limited information, you pointed that out. Fair enough!

You're still free to comment on the issue, and closing it doesn't mean It's dismissed, it's just not "active" (check #2 for instance that was kept active until resolved for the same issue you face)

Regarding your specific issue, I did link you to the relevant part of the install instructions that actually explains the very thing you initially asked about. Take a look at this image: 
![image](https://github.com/melMass/comfy_mtb/assets/7041726/dd17205e-55f4-4e3a-b258-be3afb7f4efe)


If you still have errors or issue provide more informations like what is not working for instance, as this is an optional thing (for now, the next release will have more frontend things)

Thanks
